crates/holochain_zome_types/src/block.rs

```diff
@@ -47,14 +49,14 @@ pub enum NodeBlockReason {
 
 /// Reason for a Node/Space Block.
 #[deprecated(since = "0.6.0")]
-#[derive(Clone, serde::Serialize, Debug, Eq, PartialEq, Hash)]
+#[derive(Clone, serde::Serialize, serde::Deserialize, Debug, Eq, PartialEq, Hash)]
 pub enum NodeSpaceBlockReason {
     /// Bad message encoding.
     BadWire,
 }
 
 /// Reason why we might want to block an IP.
-#[derive(Clone, serde::Serialize, Debug)]
+#[derive(Clone, serde::Serialize, serde::Deserialize, Debug)]
```

Comment:
Seems pretty unreasonable that you'd want to back the bytes you serialized... special, secret bytes!